user-stories/US-07-Privacy-request-capture-service.md

```diff
@@ -0,0 +1,300 @@
+
+### US-07-2-01
+
+**As an** Organization (not using blindnet devkit) receiving Privacy Request
+
+**I want to** know the request I received has been sent by blindnet's Privacy Request Capture Service and be informed about blindnet devkit
+
+**because** I might be interested in the service blindnet devkit offers
+
+### US-07-2-02
+
+**As an** Organization (not using blindnet devkit) receiving Privacy Request
+
+**I want to** be able to recognize the user who made the request in my system
+
+**because** so I can easily identify their data
+
+### US-07-2-03
+
+**As an** Organization, using blindnet devkit, receiving Privacy Request
+
+**I want to** receive the request in the Privacy Request section of my system
+
+**because** it is more convenient for me to have all my Privacy Requests centralized
+
+### US-07-2-04
+
+**As an** Organization (using or not using blindnet devkit) receiving Privacy Request
+
+**I want to** transfer the request to another Organization if the request concerns data I transferred to another Organization or data I got from another Organization
+
+**because** I need to be compliant, if a user wants their data deleted, I have to inform the other company I transferred the user's data to or I got the user's data from, to also delete the data
+
+## 07-3 Organization using Local Privacy Request capture service
+
+### US-07-3-01
+
+**As an** Organization, using blindnet devkit, receiving Privacy Request
+
+**I want to** have an entry point for the Privacy Request service embedded on the GDPR section of my website
+
+**because** my customers might directly go to my website to make their Privacy Request, and it is easier for my customers and for me to use this service
+
+### US-07-3-02
+
+**As an** Organization, using blindnet devkit, receiving Privacy Request
+
+**I want to** receive the Privacy Requests in a format that is compatible with my system
+
+**because** otherwise I cannot use blindnet devkit to for Privacy Request capture
+
+
+### US-07-3-03
+
+**As an** Organization, using blindnet devkit
+
+**I want to** only use some of the components, a subset of the scope of blindnet devkit for things I don't have yet
+
+**because** I want to continue using my existing interface that already covers some of my compliance needs
+
+### US-07-3-04
+
+**As an** Organization, using blindnet devkit
+
+**I want to** only use some of the components, a subset of the scope of blindnet devkit for things I don't have yet
+
+**because** I want to continue using my existing system that already covers some of my compliance needs (e.g my system already allows users to log in and modify data adresses, preferences etc - on Amazon, Facebook, etc)
+
+### US-07-3-05
+
+**As an** Organization, using blindnet devkit
+
+**I want to** use the whole blindnet devkit to automate the processing of requests
+
+**because** I don't have any system that covers my compliance needs, I do everything manually
+
+### US-07-3-06
```

Comment:
I changed the 2 US to the following, let me know what you think : 
> ### US-07-3-04
>
>**As an** Organization, using blindnet devkit
>
>**I want to** use the whole blindnet devkit components to capture the request, automate the processing of requests, view the requests...
>
>**because** I don't have any system that covers my compliance needs, I do everything manually
>
>### US-07-3-05
>
>**As an** Organization, using blindnet devkit to automate the processing of requests
>
>**I want to** automate a maximum of requests except the "other" type thant I need to process manually
>
>**because** I want to save time

Reply:
New numbering (cf. my last comment), those are now : 
>### US-07-5-02
>
>**As an** Organization, using blindnet devkit
>
>**I want to** use the whole blindnet devkit components to capture the request, automate the processing of requests, view the requests...
>
>**because** I don't have any system that covers my compliance needs, I do everything manually
>
>### US-07-5-03
>
>**As an** Organization, using blindnet devkit to automate the processing of requests
>
>**I want to** automate a maximum of requests except the "other" type of request that I need to process manually
>
>**because** I want to save time and effort
let see if we keep it as it is not in Capture service scope but also Privacy computation engine)